### PR TITLE
feat(design-system): add simple filter variant

### PR DESCRIPTION
**Pull request by ajperezbau**
EMP-311

### CODE REVIEW

File: packages/x-tailwindcss/src/x-tailwind-plugin/components/facet-filter/simple.ts [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+ * @param helpers - The {@link TailwindHelpers} to generate CSS.
+ * @returns The {@link CssStyleOptions} for the variant.
+ */
+export function facetFilterSimple(helpers: TailwindHelpers) {
+  const { theme } = helpers;
+
+  return {
+    simple: {
+      '&:hover': {
+        color: theme('colors.neutral.90'),
```
Comment (diegopf) [inline packages/x-tailwindcss/src/x-tailwind-plugin/components/facet-filter/simple.ts:15]: Currently every simple filter looks the same regardless if they have a color variant added. It is only when you select the filter then you can see something different. Is this behaviour intended?

File: packages/x-tailwindcss/demo/src/components/xds-facet-filter.vue
Comment (diegopf) [inline packages/x-tailwindcss/demo/src/components/xds-facet-filter.vue:1]: I would add at least one combination with the simple filter

File: packages/x-tailwindcss/demo/src/components/xds-facet-filter.vue [hunk 2]
```diff
@@ -97,10 +100,13 @@
...
         Ghost: this.colors.map(addParentClasses(this.base, this.ghost)),
         'Selected Ghost': this.colors.map(addParentClasses(this.base, this.ghost, this.selected)),
+        Simple: this.colors.map(addParentClasses(this.base, this.simple)),
+        'Selected Simple': this.colors.map(addParentClasses(this.base, this.simple, this.selected)),
         Disabled: [
           this.base,
           addParentClasses(this.base)(this.selected),
-          addParentClasses(this.base)(this.ghost)
+          addParentClasses(this.base)(this.ghost),
+          addParentClasses(this.base)(this.simple)
```
Comment (mnavarroespinosa) [inline packages/x-tailwindcss/demo/src/components/xds-facet-filter.vue:109]: What do you think of adding the `underline` and the `selected` variants too?

```suggestion
          addParentClasses(this.base, this.selected)(this.ghost),
          addParentClasses(this.base)(this.simple),
          addParentClasses(this.base, this.selected)(this.simple),
          addParentClasses(this.base)(this.underline),
          addParentClasses(this.base, this.selected)(this.underline)
```

Testing this change, I noticed the `disabled` styles are not properly applied on `hover` for the `underline` and the `simple+selected` variants:

image

**Review by mnavarroespinosa — CHANGES_REQUESTED** [review]

File: packages/x-tailwindcss/src/x-tailwind-plugin/theme.ts [hunk 3]
```diff
@@ -96,7 +96,7 @@ export default {
     full: '100vw'
   },
   letterSpacing: {
-    xs: '0.025em',
+    xs: '-0.025em',
```
[the remark this answers is not in the document]
Comment (ajperezbau) [reply packages/x-tailwindcss/src/x-tailwind-plugin/theme.ts:99]: Yep, it was wrong

File: packages/x-tailwindcss/src/x-tailwind-plugin/components/facet-filter/simple.ts [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+        opacity: '.6'
+      },
+      '&.selected': {
+        fontWeight: theme('fontWeight.regular'),
+        letterSpacing: theme('letterSpacing.md'),
+
+        '&:hover': {
+          color: `var(--filter-color-50, ${theme('colors.neutral.50')})`
```
[the remark this answers is not in the document]
Comment (ajperezbau) [reply packages/x-tailwindcss/src/x-tailwind-plugin/components/facet-filter/simple.ts:23]: Yep, It was outdated.
Cris just updated the designs to change it to `color-50`.

**Review by mnavarroespinosa — APPROVED** [review]

**Review by diegopf — APPROVED** [review]

**Pull request merged**